packages/pds/src/auth.ts

```diff
@@ -1,16 +1,22 @@
+import * as assert from 'node:assert'
+import { KeyObject, createPrivateKey, createSecretKey } from 'node:crypto'
+import express from 'express'
+import KeyEncoder from 'key-encoder'
+import * as ui8 from 'uint8arrays'
+import * as jose from 'jose'
 import * as crypto from '@atproto/crypto'
 import { AuthRequiredError, InvalidRequestError } from '@atproto/xrpc-server'
-import * as ui8 from 'uint8arrays'
-import express from 'express'
-import * as jwt from 'jsonwebtoken'
 import AppContext from './context'
 import { softDeleted } from './db/util'
 
 const BEARER = 'Bearer '
 const BASIC = 'Basic '
+const SECP256K1_JWT = 'ES256K'
+const HMACSHA256_JWT = 'HS256'
 
 export type ServerAuthOpts = {
   jwtSecret: string
+  jwtSigningKey?: crypto.Secp256k1Keypair
```

Comment:
is the intention to always support either symmetric or asymmetric? or just as a cutover period thing?

Reply:
In my view it would be great to support both, but it's only for cutover that we need to support them at the same time / on the same server.